src/libbson/CMakeLists.txt

```diff
@@ -473,22 +473,37 @@ endif ()
 add_subdirectory (build)
 # sub-directory 'doc' was already included above
 add_subdirectory (examples)
-add_subdirectory (fuzz)
 add_subdirectory (src)
 add_subdirectory (tests)
 
 set_local_dist (src_libbson_DIST_local
    CMakeLists.txt
    NEWS
    THIRD_PARTY_NOTICES
+   fuzz/bson.fuzz.c
+   fuzz/json.fuzz.c
 )
 
+mongoc_add_fuzzer (json-fuzz fuzz/json.fuzz.c)
+target_link_libraries(json-fuzz PRIVATE bson_static)
+
+mongoc_add_fuzzer (bson-fuzz fuzz/bson.fuzz.c)
+target_link_libraries(bson-fuzz PRIVATE bson_static)
+set_property(TARGET bson-fuzz PROPERTY FUZZER_MAX_LEN 65536)
+
+add_executable (bson2json tools/bson2json.main.c)
+add_executable (json2bson tools/json2bson.main.c)
+target_link_libraries(bson2json PRIVATE bson_static)
+target_link_libraries(json2bson PRIVATE bson_static)
+set_target_properties(bson2json json2bson PROPERTIES
+   RUNTIME_OUTPUT_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}"
+   )
+
 set (src_libbson_DIST
    ${src_libbson_DIST_local}
    ${src_libbson_build_DIST}
    ${src_libbson_doc_DIST}
    ${src_libbson_examples_DIST}
-   ${src_libbson_fuzz_DIST}
    ${src_libbson_src_DIST}
    ${src_libbson_tests_DIST}
```

Comment:
The make-release-archive task [task failure](https://spruce.mongodb.com/task/mongo_c_driver_releng_make_release_archive_patch_fe2bae72bc733515d766a46a8799783667396590_62acaf08562343599edb7070_22_06_17_16_42_49/logs?execution=0) may be resolved by including bson2json.main.c and json2bson.c in the distribution tarball.

I suggest adding a CMakeLists.txt in the src/libbson/tools directory to set the variable `${src_libbbson_tools_DIST}`.